orcid-web/src/main/resources/orcid-frontend-security.xml

```diff
@@ -313,6 +315,8 @@
 			access="IS_AUTHENTICATED_ANONYMOUSLY" />
 		<sec:intercept-url pattern="/userStatus.json(\?.*)?"
 			access="IS_AUTHENTICATED_ANONYMOUSLY" />
+		<sec:intercept-url pattern="/userInfo.json(\?.*)?"
+			access="IS_AUTHENTICATED_ANONYMOUSLY" />
```

Comment:
@amontenegro What's the reason for making this public? Is it just to make SIGN_IN_MENU available if the user isn't logged in? If that's the case, I'd suggest just removing SIGN_IN_MENU and making /userInfo.json accessible only for logged in users.

Reply:
Hi @lizkrznarich! resolved in the last push, now /userInfo.json requires an authenticated user and, the code checks if userInfo map is empty, if so, it shows the login menu.